src/GCDTracker.cs

```diff
@@ -70,7 +71,8 @@ public unsafe class GCDTracker : IDalamudPlugin
             PluginInterface.UiBuilder.OpenConfigUi += OpenConfig;
             PluginInterface.UiBuilder.OpenMainUi += OpenConfig;
             Framework.Update += ct.Update;
-            Framework.Update += gcd.Update;
+            gcdUpdateDelegate = new IFramework.OnUpdateDelegate((framework) => gcd.Update(framework, config));
```

Comment:
done

Reply:
Actually, I'm getting a compile error.  I can't get the syntax correct, VS Code complains about not having an overload IFramework.OnUpdateDelegate.

If you know how to fix this so that it works by all means.  I was actually surprised it worked as I had it.

Reply:
Sure! I'll give it a try once it's merged.